tests/framework/installer/ceph_installer.go

```diff
@@ -323,8 +323,29 @@ func (h *CephInstaller) CreateRookExternalCluster(externalManifests CephManifest
 		return errors.Wrap(err, "failed to start toolbox on external cluster")
 	}
 
-	logger.Info("Rook external cluster started")
-	return err
+	var clusterStatus cephv1.ClusterStatus
+	for i := 0; i < 8; i++ {
```

Comment:
How many retries do we typically need? Just wondering where 8 came from. :)

Reply:
3 to 4 but I got more room for it :)